<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Home</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="ppl.html">People</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="pub.html">Publications</a></div>
<div class="menu-item"><a href="ads.html">Positions</a></div>
<div class="menu-item"><a href="news.html">News</a></div>
<div class="menu-item"><a href="contact.html">Contacts</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Home</h1>
</div>
<table class="imgtable"><tr><td>
<img src="DSC_0120.JPG" alt="Golnaz Vahedi" width="170px" height="175px" />&nbsp;</td>
<td align="left"><p>Assistant Professor <br /> <a href="http://www.med.upenn.edu/genetics/">Department of Genetics</a> <br />
<a href="http://www.med.upenn.edu/ifi/">Institute for Immunology</a> <br />
<a href="http://www.med.upenn.edu/">Perelman School of Medicine</a><br />
<a href="http://www.upenn.edu/">University of Pennsylvania</a><br />
vahedi at mail.med.upenn.edu</p>
</td></tr></table>
<a href="http://twitter.com/golnaz_v" class="twitter-follow-button"
data-show-count="false">Follow @golnaz_v</a>
<script src="http://platform.twitter.com/widgets.js"
type="text/javascript"></script>
<p><a href="https://www.linkedin.com/pub/golnaz-vahedi/4/674/5a5">
<img src="http://www.linkedin.com/img/webpromo/btn_viewmy_160x33.png" width="120" height="25" border="0" alt="LinkedIn"></a></p>
<h2>Welcome</h2>
<p>The Vahedi laboratory is multidisciplinary, integrating computational and experimental approaches to develop a single to collective cell understanding of gene regulation in immune cells in health and disease. </p>
<p>What do we do? </p>
<p>We exploit the epigenomics mapping of immune cells to understand the biological circuits that underlie immune responses and uncover the molecular basis of major inherited diseases mediated by these cells. Immune-mediated disorders such as psoriasis and type 1 diabetes result from a complex interplay of genetic and environmental factors. By mapping the epigenomic alterations associated with immune-mediated diseases, we aim to further our understanding of the role of environment in trigering autoimmunity.</p>
<p>Why epigenomics? </p>
<p>Information encoded in DNA is interpreted, modified, and propagated as chromatin. The diversity of inputs encountered by immune cells demands a matching capacity for transcriptional outcomes provided by the combinatorial and dynamic nature of epigenetic processes. Advances in genome editing and genome-wide analyses have revealed unprecedented complexity of chromatin pathways involved in the immune response, offering explanations to long-standing questions and presenting new challenges. </p>
<p>How do we do research?</p>
<p>We blend epigenomics, human genetics, immunology, and computational biology to pursue a new understanding of human immunology. We generate genome-wide maps of chromatin in relevant immune cells mostly T cells. We are interested in regulators of T cell development and also T cell engagement in autoimmune disorders such as psoriasis and type 1 diabetes. We use population-based assays with strong signal-to-noise ratios such as ChIP-seq, ATAC-seq, and RNA-seq in addition to cutting-edge single-cell assays such as single-cell (sc)ATAC-seq and scRNA-seq. As a result of our computational expertise, we also harvest the vast troves of big data that immunologists and other researchers are pouring into public repositories. 
Our data integrations rely on available computational pipelines. Furthermore, we develop novel computational techniques to fully understand the complexity of multidimensional epigenomics datasets in T cells. </p>
<h2>Rotation Projects 2016-2017. </h2>
<p>1) T cell fate and clonality inference from single-cell transcriptomes. Computational and experimental.</p>
<p>2) Dissecting cellular heterogeneity in T cells using single-cell chromatin accessibility maps and single-cell transcriptomics. Computational and experimental.</p>
<p>3) Novel tools to analyze single-cell chromatin accessibility maps (scATAC-seq). Computational. </p>
<p>4) Epigenomic mapping of psoriasis. Computational and experimental.</p>
<p>5) Epigenomic mapping of type 1 diabetes. Computational and experimental.</p>
<p>6) Discovery and validation of pioneer factors in immune cells. Computational and experimental.</p>
<h2>Research Highlights </h2>
<p>The Vahedi laboratory is equipped with an automated microfluidic
solution for single-cell genomics. We are able to capture single T cells (x96) using this platform; 
our preliminary data support the feasibility of generating genome-scale chromatin accessibility maps (single-cell ATAC-seq) 
in single T cells.</p>
<table class="imgtable"><tr><td>
<img src="singlecell.jpg" alt="" width="800px" height="296px" />&nbsp;</td>
<td align="left"></td></tr></table>
<div class="infoblock">
<div class="blockcontent">
<p>You are welcome to join our weekly <a href="https://github.com/VahediLab/ComputationalJournalClub/blob/master/Schedule.md">computational journal club</a> on Fridays at 5pm (305 BRB).</p>
</div></div>
<div id="footer">
<div id="footer-text">
Page generated 2016-11-28 16:56:13 EST, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
